mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -1430,4 +1428,14 @@
     <string name="top_sites_max_limit_content">To add a new top site, remove one. Long press the site and select remove.</string>
     <!-- Confirmation dialog button text when top sites limit is reached. -->
     <string name="top_sites_max_limit_confirmation_button">OK, Got It</string>
+
+    <!-- DEPRECATED STRINGS -->
```

Comment:
Not blocking on this, but we can get rid of these in the following commit please?
CC @boek